packages/react-app/.eslintrc.json

```diff
@@ -32,6 +32,7 @@
     "@typescript-eslint/ban-ts-comment": 0,
     "@typescript-eslint/no-var-requires": 0,
     "@typescript-eslint/no-use-before-define": 0,
+    "no-mixed-spaces-and-tabs": 0,
```

Comment:
I added this due to a linting error I wasn't able to fix in a reasonable amount of time